ai_adapter.py

```
@@ -61,14 +56,14 @@ async def invoke(input: Input) -> Response:
                 {doc["source"]: doc for doc in sources}.values()
             )
 
-        return Response(json_result)
+        return Response(**json_result)
 
     except Exception as inst:
         logger.exception(inst)
```

Ruff (0.13.2)

Redundant exception object included in `logging.exception` call

(TRY401)